sql/moz-fx-data-shared-prod/monitoring_derived/duplicate_rate_v1/query.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+"""Generates counts of duplicate document_id values per table per day."""
```

Comment:
This python script is copied from the structured_missing_columns_v1 table and only lightly modified to suit this case.